src/lib.rs

```diff
@@ -2743,20 +2625,20 @@ extern "C" fn fn_generate_random(
     ret_to_rv!(get_random_data(data))
 }
 
-/// Implementation of C_GetFunctionStatus function (Not Implemented Yet)
+/// Implementation of C_GetFunctionStatus function
 ///
 /// Version 3.1 Specification: [https://docs.oasis-open.org/pkcs11/pkcs11-spec/v3.1/os/pkcs11-spec-v3.1-os.html#_Toc111203361](https://docs.oasis-open.org/pkcs11/pkcs11-spec/v3.1/os/pkcs11-spec-v3.1-os.html#_Toc111203361)
 
 extern "C" fn fn_get_function_status(_session: CK_SESSION_HANDLE) -> CK_RV {
-    CKR_FUNCTION_NOT_SUPPORTED
+    CKR_FUNCTION_NOT_PARALLEL
 }
 
-/// Implementation of C_CancelFunction function (Not Implemented Yet)
+/// Implementation of C_CancelFunction function
 ///
 /// Version 3.1 Specification: [https://docs.oasis-open.org/pkcs11/pkcs11-spec/v3.1/os/pkcs11-spec-v3.1-os.html#_Toc111203362](https://docs.oasis-open.org/pkcs11/pkcs11-spec/v3.1/os/pkcs11-spec-v3.1-os.html#_Toc111203362)
 
 extern "C" fn fn_cancel_function(_session: CK_SESSION_HANDLE) -> CK_RV {
-    CKR_FUNCTION_NOT_SUPPORTED
+    CKR_FUNCTION_NOT_PARALLEL
```

Comment:
I think for these two it might make sense to add a comment that these are legacy function really not doing anything useful. From reading the code, it is clear, but it might not be from reading just the doc text. Some note like this could do instead of the previously mention of  "not Implemented".

> Legacy function. Always returns `CKR_FUNCTION_NOT_PARALLEL`